lib/client.ts

```diff
@@ -102,10 +103,60 @@ export default class Client {
     return this.post(URL.leaveRoom(roomId));
   }
 
+  public getRichMenu(richMenuId: string): Promise<any> {
+    return this.get(URL.richMenu(richMenuId));
+  }
+
+  public createRichMenu(richMenu: Types.RichMenu): Promise<any> {
+    return this.post(URL.richMenu(), richMenu);
+  }
+
+  public deleteRichMenu(richMenuId: string): Promise<any> {
+    return this.delete(URL.richMenu(richMenuId));
+  }
+
+  public getUserRichMenuIds(userId: string): Promise<any> {
+    return this.get(URL.userRichMenu(userId));
+  }
+
+  public linkRichMenuWithUser(
+    userId: string,
+    richMenuId: string,
+  ): Promise<any> {
+    return this.post(URL.userRichMenu(userId, richMenuId));
+  }
+
+  public unlinkRichMenuWithUser(
+    userId: string,
+    richMenuId: string,
+  ): Promise<any> {
+    return this.delete(URL.userRichMenu(userId, richMenuId));
+  }
+
+  public getRichMenuContent(richMenuId: string): Promise<any> {
+    return this.stream(URL.richMenuContent(richMenuId));
+  }
+
+  public uploadRichMenuContent(
```

Comment:
I've discussed with other bot dev members, and the method name is decided to be `uploadRichMenuImage`.